ihatemoney/static/js/graphs.js

```diff
@@ -0,0 +1,132 @@
+function createGraphs(members_stats, monthly_stats) {
+    let names = [];
+    let balance = [];
+    let colors = [];
+    for (let i = 0; i < members_stats.length; i++) {

```

Comment:
We could probably use a [`for…in` statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Loops_and_iteration#for...in_statement) instead.